package.json

```diff
@@ -69,7 +69,12 @@
     "@sideway/formula": "^3.0.1",
     "@types/react": "17.0.47",
     "**/@angular-devkit/build-angular/minimatch": "3.0.5",
+    "**/@angular-devkit/build-angular/semver": "7.5.4",
+    "**/@angular/cli/semver": "7.5.4",
```

Comment:
Is t possible to bump angular directly instead? https://github.com/angular/angular-cli/commit/bd396b65623fb0b8e826be13f88709e87b54336e